### ISSUE TITLE
Tapping SpeedDial quickly twice leaves "ghost" SpeedDialChilds in view

### ISSUE THREAD

**Issue by kinex** [darioielardi/flutter_speed_dial#4]
Steps to reproduce:
1. tap SpeedDial 
2. tap SpeedDial again before "opening" animation started in step 1 is finished

Expected result:
SpeedDial is in closed state

Actual result:
SpeedDial is not closed correctly. Some of the SpeedDialChilds are left in view.

Issue may be easier to reproduce with several SpeedDialChilds (e.g. 5 or more) for that opening animation takes longer.

**Comment by darioielardi** [comment]
That's quite strange, actually I forgot to specify that it's supposed to work with at most 5 children ( as material design guidelines suggest ), and that's the actual behavior for what I can see. Maybe you're on an old version?

**Comment by kinex** [comment]
I am using flutter_speed_dial 1.0.8 (Flutter 0.8.2) and testing in a device (Galaxy S7).

I can reproduce the issue easily with 5 SpeedDialChilds, just tapping the SpeedDial repeatedly many times. With 4 SpeedDialChilds I saw also some strange behavior, but I had to try many times.

It would be nice to allow also more than 5 children if someone really needs it. In some cases breaking a guideline may be a better option than replacing the whole control with some otherwise worse optional layout.
